tests/unit/inference/test_inference.py

```diff
@@ -383,6 +383,35 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+@pytest.mark.seq_inference
```

Comment:
```suggestion
@pytest.mark.inference
```

This can be run in parallel